python/arcticdb/__init__.py

```diff
@@ -1,10 +1,14 @@
-import traceback
+import arcticdb_ext as _ext
 import os
+import sys as _sys
+import google.protobuf as _protobuf
 
-try:
-    import arcticdb_ext
-except Exception as e:
-    traceback.print_exc()
+_proto_ver = _protobuf.__version__.split(".")[0]
+if _proto_ver in "34":
+    # Use the namespace package (https://peps.python.org/pep-0420) feature to select the pb2 files matching the protobuf
+    _sys.path.append(os.path.join(os.path.dirname(__file__), "proto", _proto_ver))
+else:
+    raise NotImplementedError(f"We do not support protobuf {_protobuf.__version__}")
```

Comment:
`ArcticDB only supports protobuf version 3 or 4. You have protobuf version {_protobuf.__version__}.`.